### ISSUE TITLE
personal static files is not served

### ISSUE THREAD

**Issue by Tushar19varshney** [IMGIITRoorkee/omniport-backend#60]
**Is your feature request related to a problem? Please describe.**
As static files of the file manager are created in a folder named `personal_files`, but this folder is not served in the backend.

**Describe the solution you'd like**
In `omniport/omniport/urls/urls.py`
```python
   .....
  http_urlpatterns += static(
        settings.PERSONAL_URL,
        document_root=settings.PERSONAL_ROOT
  )
```

**Want to take up?**
Yes

**Comment by pradumangoyal** [comment]
Can't we make this folder in the `static` or `media` folder in the app itself?

**Comment by dhruvkb** [comment]
We made this a different folder because media does not involve permissions. Media uploaded by one is accessible to all users. Personal files are passed through the `X-Accel-Redirect` header pipeline to only allow the uploader to access a file.

**Comment by dhruvkb** [comment]
@rheaparekh must know the exact implementation of permission/authentication-based redirects. I remember mentioning `X-Accel-Redirect` but I never checked in on her implementation of this feature.

**Comment by Tushar19varshney** [comment]
@dhruvkb Yes, @rheaparekh implemented this feature in django_filemanager but it is not working. So, I think this issue should be shifted to django_filemanager repo.
